libs/core/src/divider/divider.element.styles.ts

```diff
@@ -0,0 +1,14 @@
+import { css } from 'lit';
+
+export default css`
+  hr {
+    margin-top: 4px;
+    margin-bottom: 4px;
+    border: 0;
+    border-top: 1px solid var(--kirby-divider-color, #ccc);
```

Comment:
`--kirby-divider-color` is part of our design tokens and should always be defined, so no need for fallback. If fallback turns out to be strictly needed, I think something from the existing palette would be better.  

```suggestion
    border-top: 1px solid var(--kirby-divider-color);
```